### PR TITLE
feat: gitpay team #537

### LINKED ISSUE #537: Gitpay team
A new page (should be included in the top menu "Team"), with a list of team members from the core. The data can be a simple JSON with name, role, description, picture URL, website, and Github (or Bitbucket) profile.

A component to notify someone who wants to join the team. The join action should send an email to our notifications mail (in a constants file) with the content:

```
The user with the email {{email}} wants to join to Gitpay core team
```

### Why
We have a core team and we need to display who is working for Gitpay.

### Screenshot attached
gitpay-team

### Assets
Group2

### PR DESCRIPTION
**Pull request by salonigoyal2309**
## Description

> 1. '/team' route renders core team page ( as provided in the issue screenshot)
> 2. TeamCard.js is generalized 
    'display : flex' was keeping all cards in same row by shrinking them
    fix: some minor css changes

## Screenshot
screencapture-localhost-8082-2020-09-20-19_33_10

### CODE REVIEW

**Comment by salonigoyal2309** [comment]
@alexanmtz hey I tried my best to resolve the issue , please review my changes

**Comment by alexanmtz** [comment]
Hey @salonigoyal2309, that's great to see the result. I think before going into the code review, there are some minor bugs and improvements that I will ask you to update:

1. For the Github button, if we make it black and with Github icon?
2. The text "join the team" and "Work with the best and be part of the core" should have some style, you can see it is in a generic font, not the default font, so probably you need to use the Typography component from Material-ui:
https://material-ui.com/components/typography/

So please make these changes in order to proceed.

**Comment by salonigoyal2309** [comment]
@alexanmtz How about this ? if there is anything else required please let me know , if not i'll commit changes
screencapture-localhost-8082-2020-09-21-14_06_49

**Comment by alexanmtz** [comment]
Hey, @salonigoyal2309 this is great, and thanks for the update.

Now I just see one last small issue. Can you see that the Github icon there's no space like LinkedIn? Can you check the space to keep the same as the Linkedin Icon? You can check how is done there, so probably it is an easy fix.

**Comment by salonigoyal2309** [comment]
@alexanmtz i have done all the changes as requested :D
screencapture-localhost-8082-2020-09-21-16_36_37

**Comment by alexanmtz** [comment]
Great @salonigoyal2309, so let's review the code

File: frontend/src/components/team/team.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    }
+    axios
+      .post(api.API_URL + '/team/core/join', { param })
+      .then(res => {
+        setStatus('Email sent')
+      })
+      .catch(err => {
+        setStatus('Oops! Error Happended' + err)
+      })
+  }
```
Comment (alexanmtz) [inline frontend/src/components/team/team.js:100]: @salonigoyal2309 please move this to actions and reducers, see how we do in the rest of the app to follow the same pattern.

**Review by alexanmtz — CHANGES_REQUESTED** [review]
@salonigoyal2309 great 🎉 

We just need a refactoring to use actions and reducers to make the requests. Are you able to do this?

File: frontend/src/components/team/team.js (hunk 1 above)
Comment (salonigoyal2309) [reply frontend/src/components/team/team.js:100]: @alexanmtz sure , let me give it a try 👍

File: modules/team/requestJoinCoreTeam.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const Promise = require('bluebird')
+const SendMail = require('../mail/mail')
+const { notificationEmail } = require('../mail/constants')
+
+module.exports = Promise.method(function requestJoinCoreTeam (params) {
+  SendMail.success(
+    { email: notificationEmail, language: 'en' },
```
Comment (alexanmtz) [inline modules/team/requestJoinCoreTeam.js:7]: We need to create a new email notification here, case I'm receiving an error, cause the notification email is already as a cc.

So can you add to `constants` file a something like:
```
joinTeamEmail: "team@gitpay.me"
``` 
So you can import and use like this

```suggestion
    { email: joinTeamEmail, language: 'en' },
```

**Review by alexanmtz — CHANGES_REQUESTED** [review]
@salonigoyal2309 great, thanks for the update.

There's just one adjustment we need to do to the destination e-mail.

Another thing missing is the bottom bar, can you add to the team page as well?

**Comment by salonigoyal2309** [comment]
@alexanmtz by bottom bar do you mean this ?
bottom

**Comment by alexanmtz** [comment]
> @alexanmtz by bottom bar do you mean this ?
> bottom

Exactly, it's a component that can be imported. Sorry, it's not on the layout, but it should be present on all pages.

**Review by alexanmtz — APPROVED** [review]
Great @salonigoyal2309, now we're good to go 🎉,

thanks for your patience and you did a great job 🥇 

Soon you will see the final page in production.

**Comment by salonigoyal2309** [comment]
@alexanmtz Thanks for being patient and helping me 🎉

**Pull request merged**